server/lib/seed/seed-data.js

```diff
@@ -185,6 +185,14 @@ export const pages = [{
 <p>Thanks for your donations to:</p>
 <p><span class="ql-placeholder-content" data-id="receipt" data-label="Donation Receipt" data-required="true"></span></p>
 `
+}, {
+  identifier: pageIdentifiers.GOOGLE_SINGUP,
+  title: 'Google Sined Up',
+  type: pageTypes.EMAIL,
+  subject: '<p>The User Sined Up with Google!</p>',
```

Comment:
This goes to users who think they forgot their password but can actually just sign in with google. They don't want to be called `The User`! I'd put something like:
`<Foodbank Name>` password reset request
Hi `<User Name>`,
A request was made to reset your password for `<Foodbank Name>` but your account is linked to your google account. You can just login by clicking the 'Sign in with Google' button.
If you still cannot login you can email our support team at `<Foodbank Support Email>`
If you didn't make this request you can ignore this email